udata_hydra/cli.py

```diff
@@ -155,10 +155,11 @@ async def check_resource(resource_id: str, method: str = "get"):
 
 @cli(name="analyse-csv")
 async def analyse_csv_cli(
-    check_id: int | None = None, url: str | None = None, debug_insert: bool = False
+    check_id: str | None = None, url: str | None = None, debug_insert: bool = False
```

Comment:
Hmm is this a minicli issue? It wouldn't cast it to `int` due to `| None`?

Reply:
No I don't think it was a minicli issue. But all arguments of minicli are strings by default, so it needs to be casted as int (which I did on the fist line of the method). Type hinting argument cannot cast or change the type, it was useless (and false).

Reply:
I think `size` in `csv_sample` command is used as int directly?

Reply:
Good point, my guess is the type hint is wrong, and it's indeed being manipulated in the code as a string.
We should check and fix that.

Reply:
OK, just double-checked, you are right, it's seems to be related to the `int | None`, which seems to changes minicli's casting behaviour. It seems to be a bug from minicli. We already had a very similar bug before. I will created a ticket.

Reply:
OK thanks for spotting, I just created an issue: https://github.com/pyrates/minicli/issues/15

In the meantime we can continue using hydra CLI as it is.